src/pudl_archiver/archivers/ferc/ferceqr.py

```diff
@@ -0,0 +1,68 @@
+"""Download FERC EQR data."""
+from pathlib import Path
+
+from pudl_archiver.archivers.classes import (
+    AbstractDatasetArchiver,
+    ArchiveAwaitable,
+    ResourceInfo,
+)
+
+BASE_URL = "https://eqrreportviewer.ferc.gov/"
+
+
+class FercEQRArchiver(AbstractDatasetArchiver):
+    """FERC EQR archiver."""
+
+    name = "ferceqr"
+
+    async def get_resources(self) -> ArchiveAwaitable:
```

Comment:
It *might* be easier to test this if you have something that generates all the links, then something else that downloads all of them. Then you could have a test that just checks that all the links that were generated can be downloaded with a `HEAD` request [or something](https://stackoverflow.com/questions/16778435/python-check-if-website-exists).